app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(117, 4)) == 117
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    for element in split_integer(12, 4):
+        assert element == 3
```

Comment:
Instead of iteration, perform `assert` directly to a list as it is done in `test_should_add_zeros_when_value_is_less_than_number_of_parts`.